datasets/simulated.py

```diff
@@ -11,11 +11,8 @@ class Dataset(BaseDataset):
     # List of parameters to generate the datasets. The benchmark will consider
     # the cross product for each key in the dictionary.
     parameters = {
-        'n_samples, n_features': [
-            (100, 5_000),
-            (100, 10_000)
-        ],
-        'rho': [0, 0.6],
+        "n_samples, n_features": [(100, 5_00), (100, 10_0)],
```

Comment:
unintended change

Reply:
sorry, I probably did a commit -am "..." without realising.
Undoing it right now